modules/node_modules/@ncigdc/utils/filters/index.js

```diff
@@ -81,7 +81,7 @@ export const replaceFilters: TMergeFilters = (q, ctxq) => {
   const merged = {
     op: 'and',
     content: ctxq.content.reduce((acc, x) => {
```

Comment:
PR open
thanks for catching this
https://github.com/NCI-GDC/portal-ui/pull/1708/files